src/sentry/seer/similarity/utils.py

```diff
@@ -446,3 +476,55 @@ def set_default_project_seer_scanner_automation(
         project.update_option(
             "sentry:default_seer_scanner_automation", org_default_seer_scanner_automation
         )
+
+
+def get_token_count(
```

Comment:
i'm wondering if we should put this behind an option for the initial deploy just to be safe?

Reply:
oh woops, i replied to the wrong comment. i mean just around running the function to report the metric, in case for whatever reason there's a problem when we deploy and need to roll back. (thinking perf/memory/whatever problem)